### PR TITLE
Addition of HybridDataManager 

### PR DESCRIPTION
**Pull request by ghost**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json
Comment (jianghaolu) [inline specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json]: The 200 response model for an ARM PUT operation must have x-ms-azure-resource extension set to true in its hierarchy

File: specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json
Comment (jianghaolu) [inline specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json]: The 200 response model for an ARM PUT operation must have x-ms-azure-resource extension set to true in its hierarchy

File: specification/hybriddatamanager/resource-manager/readme.md
Comment (jianghaolu) [inline specification/hybriddatamanager/resource-manager/readme.md]: Why did you delete the suppressions for R2016 and R2062? They are failing in lintdiff now.

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/hybriddatamanager/resource-manager/readme.md
Comment (t-prta) [reply specification/hybriddatamanager/resource-manager/readme.md]: The suppressions were added by mistake without approval from the review team. Once we get the approval for the suppressions, I will add the required directives again.

File: specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json
Comment (t-prta) [reply specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json]: Waiting for a suppression on this issue.

File: specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json
Comment (t-prta) [reply specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json]: Waiting for a suppression on this issue.

**Comment by dsgouda** [comment]
startbuild

**Comment by jianghaolu** [comment]
@t-prta Please let us know when you are planning to update the suppressions. Thank you!

File: specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json
Comment (t-prta) [reply specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json]: Suppression added

File: specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json
Comment (t-prta) [reply specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json]: Suppression added

File: specification/hybriddatamanager/resource-manager/readme.md
Comment (t-prta) [reply specification/hybriddatamanager/resource-manager/readme.md]: Suppression added

**Review by anuchandy — APPROVED** [review]
2 ARM suppressions added.  ARM team needs to follow up on this and if suppressions are still a problem service team will have to sync on this and take necessary action (including reverting them).

**Comment by anuchandy** [comment]
Ruby, NET failures are expected since this is a new package. 

@shahabhijeet  @sarangan12 we're taking this PR, please onboard this new RP.

**Pull request merged**
